van-gogh-second-page/jons/index-tickets.css

```diff
@@ -0,0 +1,125 @@
+@import url('https://fonts.googleapis.com/css2?family=Varela+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@100;400;700&display=swap');
```

Comment:
Let's remove this font if you don't need it

Reply:
I use "Varela" but I´ve removed the others